src/main/java/org/spongepowered/common/data/nbt/validation/SpongeEntityValidator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.data.nbt.validation;
+
+import net.minecraft.nbt.NBTTagCompound;
+import org.spongepowered.api.data.DataView;
+
+public class SpongeEntityValidator implements RawDataValidator {
+
+    @Override
+    public ValidationType getValidationType() {
+        return Validations.ENTITY;
+    }
+
+    @Override
+    public boolean validate(NBTTagCompound view) {
+        // TODO implement this bit, the view needs various things, but it most definteily cannot
```

Comment:
The Data implementation for schematics of manipulating raw not is not going to be implemented just yet as it is not a requirement for merging Schematics. However, it is nonetheless a system defined to allow for further implementation once the branch is merged.